CRM/Import/Parser.php

```diff
@@ -1570,23 +1586,21 @@ protected function getTransformedFieldValue(string $fieldName, $importedValue) {
       $comparisonValue = $this->getComparisonValue($importedValue);
       return $options[$comparisonValue] ?? 'invalid_import_value';
     }
-    if (!empty($fieldMetadata['FKClassName']) || ($fieldMetadata['pseudoconstant']['prefetch'] ?? NULL) === 'disabled') {
```

Comment:
This if doesn't add anything as the fields inside it are hard-coded to specific fields anyway.....